stock_location_address_purchase/tests/test_purchase_address.py

```diff
@@ -54,10 +54,10 @@ def test_onchange_purchase(self):
         purchase = self.env["purchase.order"].new(
             {"partner_id": self.partner.id, "picking_type_id": self.picking_01.id}
         )
-        purchase._onchange_picking_type_id()
+        purchase._compute_dest_address_id()
```

Comment:
If it is a compute, you don't need to specify the execution

Same for other executions